examples/ae_examples/cvae_examples/conv_cvae_example/models.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class ConvConditionalEncoder(nn.Module):
+    def __init__(
+        self,
+        latent_dim: int,
+        condition_vector_size: int = 2,
+    ) -> None:
+        super().__init__()
+
+        self.conv = nn.Sequential(
+            nn.Conv2d(1, 8, 5, stride=2),  # 12*12
+            nn.ReLU(),
+            nn.Conv2d(8, 16, 5, stride=2),  # 4*4
+            nn.ReLU(),
+        )
+        self.fc1 = nn.Linear(16 * 4 * 4 + condition_vector_size, 64)
+        self.fc_mu = nn.Linear(64, latent_dim)
+        self.fc_logvar = nn.Linear(64, latent_dim)
+
+    def forward(self, input: torch.Tensor, condition: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+        x = self.conv(input)
+        # Flatten the tensor
+        x = x.view(x.size(0), -1)
+        # Condition is a one_hot encoded vector
+        x = torch.cat((x, condition), dim=-1)
+        x = F.relu(self.fc1(x))
+        return self.fc_mu(x), self.fc_logvar(x)
+
+
+class ConvConditionalDecoder(nn.Module):
+    def __init__(
+        self,
+        latent_dim: int,
+        condition_vector_size: int = 2,
+    ) -> None:
+        super().__init__()
+
+        self.fc4 = nn.Linear(latent_dim + condition_vector_size, 64)
+        self.fc5 = nn.Linear(64, 16 * 4 * 4)
+        self.deconv = nn.Sequential(
+            nn.ConvTranspose2d(16, 8, 5, stride=2),  # 11*11
```

Comment:
I think the comments on these lines are output dimensions? If so maybe just add that for clarity?